README.md

```diff
@@ -1,4 +1,8 @@
-# aqua
+<p align="center" width="100%">
+<img src="./logo/aqua_horizontal.svg" width="400">
```

Comment:
You can check how it looks in the README [here.](https://github.com/suzuki-shunsuke/aqua/blob/3440f684fc9e782ea5adf2f2d4ad613fa0657f45/README.md)